server/src/test/java/com/hedera/block/server/grpc/BlockAccessServiceTest.java

```diff
@@ -64,38 +60,29 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.io.TempDir;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 
 @ExtendWith(MockitoExtension.class)
 class BlockAccessServiceTest {
 
-    @Mock private Notifier notifier;
-
     @Mock private StreamObserver<SingleBlockResponse> responseObserver;
 
-    @Mock private LiveStreamMediator streamMediator;
-
     @Mock private BlockReader<Block> blockReader;
 
     @Mock private BlockWriter<List<BlockItem>> blockWriter;
 
     @Mock private ServiceStatus serviceStatus;
 
-    private final System.Logger LOGGER = System.getLogger(getClass().getName());
-
-    private static final String TEMP_DIR = "block-node-unit-test-dir";
-
     private static final int testTimeout = 1000;
 
-    private Path testPath;
+    @TempDir private Path testPath;
```

Comment:
We do not need cleanup in the `@AfterEach` method, by default junit will attempt to recursively delete the `@TempDir` and if it does not succeed it should throw an exception and the test should fail (this should be based on documentation). Consider to take a look if we could remove this teardown logic and if indeed everything cleans up without it by default.